R/html-tweak.R

```diff
@@ -109,6 +113,9 @@ tweak_tables <- function(html) {
   invisible()
 }
 
+#' @noRd
+#' @param x attributes of an HTML node
+#' @param class class to be appended (to pre-existing class, if any)
```

Comment:
I think the root cause of this issue is that the interface of `prepend_class` is surprising —  why doesn't it work with an xml_node object? Could you take a look into what fixing that would involve?